### PR TITLE
Radial bar chart - Use colors from palette

### PR DESCRIPTION
**Pull request by BugsBunny338**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: radial_bar_chart_plugin/src/dp_radial_bar_chart_plugin/RadialBarChart.jsx [hunk 1]
```diff
@@ -19,6 +26,8 @@ const RadialBarChart = (props) => {
         ErrorComponent,
         widget,
     } = props;
+    const colorPalette = useDashboardSelector(selectColorPalette).map(color =>
```
Comment (kandl) [inline radial_bar_chart_plugin/src/dp_radial_bar_chart_plugin/RadialBarChart.jsx:29]: Should it be memoized?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: radial_bar_chart_plugin/src/dp_radial_bar_chart_plugin/RadialBarChart.jsx (hunk 1 above)
Comment (BugsBunny338) [reply radial_bar_chart_plugin/src/dp_radial_bar_chart_plugin/RadialBarChart.jsx]: Done!

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-plugin-examples-gate-PR33

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Pull request merged**